mainDoc.html

```diff
@@ -0,0 +1,544 @@
+            </li>
+            <li class="nav-ul-item">
+                <a class="nav-link" href="#">Documentation</a>
+            </li>
+            <li class="nav-ul-item"><a class="nav-link" href="https://github.com/saisumanthkacham/Dunzo-UI">GitHub</a></li>
+        </ul>
+        <div class="nav-item">
+            Version 1.0
+        </div>
+    </nav>
+    <aside>
+        <ul class="aside-ul">
+            <li class="aside-ul-items"><a class="link" href="#avatar">Avatar</a></li>
+            <li class="aside-ul-items"><a class="link" href="#alert">Alert</a></li>
+            <li class="aside-ul-items"><a class="link" href="#badge">Badge</a></li>
+            <li class="aside-ul-items"><a class="link" href="#button">Button</a></li>
+            <li class="aside-ul-items"><a class="link" href="#card">Card</a></li>
+            <li class="aside-ul-items"><a class="link" href="#input">Input</a></li>
+            <li class="aside-ul-items"><a class="link" href="#modal">Modal</a></li>
+            <li class="aside-ul-items"><a class="link" href="#navbar">Navbar</a></li>
+            <li class="aside-ul-items"><a class="link" href="#text">Text</a></li>
+        </ul>
+    </aside>
+
+
+
+
+
+
+
+
+    <main class="main ">
+
+        <!-- avatar -->
+        <div class="avatar " id="avatar">
+
+            <h2>⚡Avatars</h2><br>
+            <p class="des-txt">In computing, an avatar (also known as a profile picture or userpic) is a graphical representation of a user or the user's character or persona</p>
+            <section>
+                <h3>>> Image Avatar</h3><br>
+                <img src="images/girl.svg" alt="" class="ava ava-md ">
+                <img src="images/woman.svg" alt="" class="ava ava-md">
+                <img src="images/an.svg" alt="" class="ava ava-md">
+
+                <br><br>
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=lucario&wt=none&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=13px&lh=139%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%253E%250A%2520%2520%253Cimg%2520src%253D%2522images%252Fgirl.svg%2522%2520alt%253D%2522%2522%2520class%253D%2522ava%2520ava-md%2520%2522%253E%250A%2520%2520%253Cimg%2520src%253D%2522images%252Fwoman.svg%2522%2520alt%253D%2522%2522%2520class%253D%2522ava%2520ava-md%2522%253E%250A%2520%2520%253Cimg%2520src%253D%2522images%252Fan.svg%2522%2520alt%253D%2522%2522%2520class%253D%2522ava%2520ava-md%2522%253E%250A%2520%253C%252Fdiv%253E"
+                    style="width: 505px; height: 134px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <br>
+            </section>
+
+
+            <section>
+                <h3>>> Icon Avatar</h3><br>
+                <div class="row ">
+                    <div class="icon ter-bg "><i class="fab fa-snapchat-ghost fa-2x white-font"></i>
```

Comment:
```suggestion
                    <div class="icon tertiary-bg "><i class="fab fa-snapchat-ghost fa-2x white-font"></i>
```
Use the same naming convention